app/frontend/src/App.tsx

```diff
@@ -35,10 +43,28 @@ function App() {
         <ReactQueryClientProvider>
           <AuthProvider>
             <CssBaseline />
-            <Navigation />
-            <Container maxWidth="md" className={classes.padding}>
-              <AppRoutes />
-            </Container>
+            <Switch>
+              <Route path={authRoute}>
+                <Container
+                  maxWidth="md"
+                  className={classes.fullscreenContainer}
+                >
+                  <AuthPage />
+                </Container>
+              </Route>
+              <Route path={`${loginRoute}/:urlToken?`}>
+                <Login />
+              </Route>
+              <Route path={`${signupRoute}/:urlToken?`}>
+                <Signup />
+              </Route>
+              <Route path="/">
+                <Container maxWidth="md" className={classes.standardContainer}>
+                  <Navigation />
+                  <AppRoutes />
+                </Container>
+              </Route>
+            </Switch>
```

Comment:
This should happen in AppRoutes.tsx. I think a more extensible way is to make an `AppRoute` component with props that extends `RouteProps` from `react-router-dom` with two extras - `private` and `fullScreen`. `private` makes it require login (which is what `Privateroute` is doing now) and `fullScreen` decides if it's in the container or not.

Reply:
(This would replace `PrivateRoute` and all routes would use the new component)

Reply:
done